.github/workflows/run-checks.yml

```diff
@@ -0,0 +1,26 @@
+name: Run checks
+
+on:
+  pull_request:
+    branches:
+    - main
+
+jobs:
+  
+  run-checks:
+    # runs-on: macOS-latest
+    runs-on: self-hosted
+    steps:
+    
+    - name: Checkout
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 1
+    
+    - name: Install Dependencies
+      run: |
+        brew install mint
+        mint install NickLockwood/SwiftFormat@0.48.17 --no-link
```

Comment:
HB is using v0.51.15 now, v0.48.17 is quite old

Reply:
Thanks. Now using the latest release.